src/extensions/default/analyzers/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers/BinaryFormatterUnsafeDeserializeAnalyzer.cs

```diff
@@ -39,71 +38,81 @@ public override void Initialize(AnalysisContext context)
             context.EnableConcurrentExecution();
             context.ConfigureGeneratedCodeAnalysis(GeneratedCodeAnalysisFlags.Analyze | GeneratedCodeAnalysisFlags.ReportDiagnostics);
 
-            context.RegisterSyntaxNodeAction(AnalyzeMemberAccessExpressions, SyntaxKind.SimpleMemberAccessExpression);
+            context.RegisterSyntaxNodeAction(AnalyzeMemberAccessExpressions, CodeAnalysis.CSharp.SyntaxKind.SimpleMemberAccessExpression);
+            context.RegisterSyntaxNodeAction(AnalyzeMemberAccessExpressions, CodeAnalysis.VisualBasic.SyntaxKind.SimpleMemberAccessExpression);
```

Comment:
If we're going to do this as a pattern, let's create an extension method that will register both of these and handle as much book keeping as necessary.

I had put in a `RegisterMemberAccessExpression` kind of extension at some point (it operates at the symbol layer), but that kind of pattern could be used here to simplify all (regardless of symbol/syntax level).

Reply:
created an extension method. I didn't see one available already.